REQUIREMENTS.rst

```diff
@@ -0,0 +1,72 @@
+============================
+ CESM software requirements
+============================
+
+Software requirements for installing, building and running CESM
+===============================================================
+
+Installing, building and running CESM requires:
+
+* a Unix-like operating system (Linux, AIX, OS X, etc.)
+
+* git client version 1.8 or newer
+
+* subversion client version ??? or newer
+
+* python2 version 2.7 or newer
```

Comment:
I didn't list python3 because I'm not sure if we test cesm as a whole with python3 (I know cime testing works with python3, but I don't know about, e.g., a B compset). @jedwards4b do you know?

Reply:
Thus far only cime has been tested with python 3.